.env

```diff
@@ -7,7 +7,7 @@ REACT_APP_COVER_IMAGE_ALT=https://cdn.ohc.network/care_logo.svg
 REACT_PUBLIC_URL=https://care.ohc.network
 
 # Care API URL without the /api prefix
-REACT_CARE_API_URL=https://care-api.do.ohc.network
+REACT_CARE_API_URL=https://staging-api.ohc.network
```

Comment:
this would cause merge conflicts in future. We should be setting the env vars in the deployment instead of updating the `.env`

Reply:
it's already pointing to staging BE in cloudflare

![image](https://github.com/user-attachments/assets/944a0ac2-9a8b-4430-95da-9ffb538e17c2)